parsec/interfaces/ptg/ptg-compiler/jdf.h

```diff
@@ -189,6 +194,14 @@ typedef unsigned int jdf_flags_t;
 #define JDF_PROP_UD_STARTUP_TASKS_FN_NAME      "startup_fn"
 
 #define JDF_FUNCTION_HAS_UD_DEPENDENCIES_FUNS  ((jdf_flags_t)(1 << 4))
+
+#define JDF_PROP_TERMDET_USER_TRIGGERED        "user-triggered"
+#define JDF_HAS_USER_TRIGGERED_TERMDET         ((jdf_flags_t)(1 << 5))
+
+#define JDF_PROP_TERMDET_DYNAMIC               "dynamic"
```

Comment:
Same property name as JDF_PROP_DYNAMIC_JDF.

Reply:
Should be solved